client/i18n/locales/english/intro.json

```
@@ -295,11 +295,17 @@
   },
   "relational-databases": {
     "title": "Relational Databases",
-    "intro": ["placeholder"],
+    "intro": [
+      "For these courses, you will use real developer tools and software like VS Code, PostgreSQL, and the Linux / Unix command line to complete interactive tutorials and build projects."
+    ],
     "blocks": {
       "learn-relational-databases": {
         "title": "Learn Relational Databases",
-        "intro": ["placeholder"]
+        "intro": [
+          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to the advanced usage of and connection to a database.",
```

```suggestion
          "These courses start you off with basic Bash commands. Using the terminal, you will learn everything from navigating and manipulating a file system, scripting in Bash, all the way to advanced database usage.",
```
"advanced usage of and connection to" is a bit of a mouthful and I'm not sure mentioning "connection" adds much.